http/src/main/resources/reference.conf

```diff
@@ -337,6 +337,7 @@ azure {
     ]
     enabled = true
     chart-versions-to-exclude-from-updates = []
+    ecm-base-uri = "https://externalcreds.dsde-dev.broadinstitute.org"
```

Comment:
Should this be called `ecm-base-url` since that's what it is called in leo.conf? Or maybe leo.conf one should be called `emc-base-uri`?

Reply:
It looks like everywhere else was `uri` so I opted to change `leo.conf` instead